data/csp/ec2/settings/suma-proxy/sle15/sp5/packages.yaml

```diff
@@ -0,0 +1,8 @@
+packages:
+  _namespace_ec2_tools:
+    package:
+      - amazon-ssm-agent
+      - python3-ec2metadata
+  _namespace_ec2_image_utils:
+    package:
+      - python3-ec2imgutils
```

Comment:
Let's drop ec2imgutils from the SUMa images.

Reply:
Ok. Should we drop the packages from 4.3 as well? It would fix the current build issues.